cmd/loraserver/main.go

```diff
@@ -420,6 +420,11 @@ func main() {
 			EnvVar: "GW_MQTT_PASSWORD",
 		},
 		cli.StringFlag{
+			Name:   "gw-mqtt-ca-file",
```

Comment:
Could you change this to `mqtt-ca-cert` (+ also on lin 194). That way it is consistent with the other flags :-)